docs/building-images.md

```diff
@@ -46,6 +46,10 @@ operatingSystem:
     chronyServers:
       - 10.0.0.1
       - 10.0.0.2
+  proxy:
+    httpProxy: http://10.0.0.1:3128
+    httpsProxy: http://10.0.0.1:3128
+    noProxy: localhost, 127.0.0.1, edge.suse.com
```

Comment:
As you've mentioned offline, I'm not sure whether this being a single string is better than a list. Are these values usually copy pasted on a single line?

Reply:
I think this is fine as it is and we can adapt later if users would prefer a list.

Reply:
Just keep in mind that adapting later is much more complicated for something like the image definition, which represents our API. Not impossible, but I wanted to at least say that the definition is one area where we shouldn't necessarily push off too many decisions until later.